src/main/java/core/basesyntax/model/User.java

```diff
@@ -1,61 +1,68 @@
-package core.basesyntax.model;
-
-import java.util.Objects;
-
-public class User {
-    private Long id;
-    private String login;
-    private String password;
-    private Integer age;
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getLogin() {
-        return login;
-    }
-
-    public void setLogin(String login) {
-        this.login = login;
-    }
-
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
-
-    public Integer getAge() {
-        return age;
-    }
-
-    public void setAge(Integer age) {
-        this.age = age;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        User user = (User) o;
-        return Objects.equals(login, user.login)
-                && Objects.equals(password, user.password)
-                && Objects.equals(age, user.age);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(login, password, age);
-    }
-}
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class User {
+    private Long id;
+    private String login;
+    private String password;
+    private Integer age;
+    
+    public User(Long id, String login, String password, Integer age) {

```

Comment:
Consider using a constructor without the id parameter, as the id is usually generated when a user is persisted to a database. Creating a constructor without the id parameter would make it easier for users to create a new User object.